inc/functions_post.php

```diff
@@ -401,64 +401,81 @@ function build_postbit($post, $post_type=0)
 		}
 
 		// Display profile fields on posts - only if field is filled in
-		if(is_array($profile_fields))
```

Comment:
> Probably adding your if(!isset($profile_fields))check before the current if(is_array($profile_fields)) one would fix the quick reply.

No. The `build_postbit()` would still be in newreply.php. Also, the fixes you're both suggesting as an alternative would break posssible "announcement profile fields" plugins or whatever the same way, especially a new function for it.